docs/docs/warehouses/bigquery.mdx

```diff
@@ -111,6 +111,12 @@ table (which will be `experiment_viewed` if you use Segment or Rudderstack).
 
 When you click save, GrowthBook will test the connection to make sure the credentials are correct. If the connection is successful, you should see a success message on the next page.
 
+## Enabling Data Pipeline Mode (Enterprise)
+
+Enterprise customers can enable pipeline mode, which can reduce query costs if you grant the GrowthBook service account write permissions in your data warehouse.
+
+[More details can be found here.](/experimentation-analysis/data-pipeline)
```

Comment:
This should link to the slug, not the filename.  So `/app/data-pipeline`.  Same for the snowflake page

Reply:
Ah, sorry. I messed around with this last minute. Fixed now.